### DISCUSSION TITLE
Loving Version 11

### DISCUSSION THREAD

**Discussion by shanecowherd** [intitni/CopilotForXcode#144] [General]
Keep up the awesome work, I look forward to the Sparkle update screen!

The Global Chat, increased height, copy/paste buttons are winners for me!

I've been playing with keyboard shortcuts, these are my `favorites` so far.

`Shift + Tab` for Accept Suggestion
`Shift + Esc` for Reject Suggestion
`Option Space` for Chat with Selection

<img width="628" alt="Screenshot 2023-04-11 at 5 54 45 PM" src="https://user-images.githubusercontent.com/442439/231313008-848849d4-fc20-497d-9c57-7a3641f0f2bc.png">

**Comment by ICULikeMac** [comment]
Thanks for sharing! What is the 'Reject Suggestion' key after shift?

**Reply by shanecowherd** [reply]
Shift + Esc for Reject Suggestion

**Comment by ChrisMiami** [comment]
So, you have to *ask* it for a suggestion, they don’t just appear?
